fvm/handler/programs.go

```diff
@@ -110,19 +110,19 @@ func (h *ProgramsHandler) Get(location common.Location) (*interpreter.Program, b
 }
 
 func (h *ProgramsHandler) Cleanup() error {
-
 	stackLen := len(h.viewsStack)
 
 	if stackLen == 0 {
 		return nil
 	}
 
-	for i := stackLen; i > 0; i-- {
+	for i := stackLen - 1; i > 0; i-- {
 		entry := h.viewsStack[i]
```

Comment:
does that mean this line caused an index out of bounds previously?

Reply:
We never called this method before.